helm/promxy-app/templates/ingress-basic-auth.yaml

```diff
@@ -24,11 +28,19 @@ spec:
   - host: {{ .Values.monitoring.prometheus.host }}
     http:
       paths:
-      - backend:
+      - path: /
+        pathType: Prefix
+        backend:
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" }}
+          service:
+            name: {{ include "resource.default.name" . }}
+            port:
+              number: {{ .Values.promxy.port }}
+{{- else }}
           serviceName: {{ include "resource.default.name" . }}
           servicePort: {{ .Values.promxy.port }}
-        path: /promxy(/|$)(.*)
```

Comment:
@ubergesundheit Is this okay to drop?

Reply:
mh probably not. I'll have to do a follow up pr